build/amdclean.js

```diff
@@ -26,7 +26,7 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 */
 
-;(function(esprima, estraverse, escodegen, _) {
+;var sourcemapToAst;(function(esprima, estraverse, escodegen, _) {
```

Comment:
Yes, sure, it looks better, committing. Although I am not sure about the value of the typeof check for this variable... but I did it the same way as the others.